controller/watcher.go

```diff
@@ -73,7 +73,7 @@ func retryConsul(retryCount int) (bool, time.Duration) {
 	random := rand.New(rand.NewSource(time.Now().UnixNano()))
 	wait := retry.WaitTime(uint(retryCount), random)
 	dur := time.Duration(wait)
-	log.Printf("[WARN] (hcat) error connecting with Consul. Waiting %v to retry"+
+	log.Printf("[DEBUG] (hcat) error connecting with Consul. Waiting %v to retry"+
```

Comment:
For operators who have above debug-level logs configured, there will be some silence in the logs while this retries. I wonder if this could be confusing to operators? Or maybe this is expected experience for those configure above debug-level? Just curious what your thoughts are!

Reply:
Most logs are reviewed after the fact where they'd see the initialization and the final failed retry. If they're watching the logs more actively they'd probably have enabled more verbose logging.

We could bump it up to INFO level or down to TRACE level... DEBUG level seemed the best at the time. IMO INFO is one of those in-between values that never seems that useful and TRACE is usually very verbose.

Reply:
Ah, good point about operators setting the log-level depending on if they're looking after the fact or actively! Thanks for distinguishing those two and sharing your thoughts. It was helpful for me to think through. I agree with you that DEBUG seems to be the best level.